src/index.json

```diff
@@ -3480,6 +3480,52 @@
                 "sha256Digest": "bbf330942daac0035bd9bb9a0fdeaf297124ec8cd98e28e46adb2bf34070295b"
             }
         ],
+        "baremetal": [
```

Comment:
Please rename this to something like `baremetal-infrastructure` to be specific

Reply:
Will do, but I believe that I will need to make a new release on our CLI extension to reflect this naming change

Reply:
I have updated in both places now, so it should be good